solutions/monitoring_platforms_intro/index.asciidoc

```diff
@@ -0,0 +1,131 @@
+//Platform=Azure
+//Maturity level=Advanced
+
+:toc: macro
+toc::[]
+:idprefix:
+:idseparator: -
+
+== Monitoring
+=== Context & Problem
+==== Terminology
+
+_Used definition: A monitoring solution helps the monitoring consumer achieve the satisfactory level of control of a defined service_. (https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/manage/monitor/observability[Link])
+
+This definition already includes the following:
+
+* *Defined service:* The resources you want to monitor aka monitored resources. The resources to be monitored can be split in infrastructure and applications on top. 
+* *Level of control:* That is your bandwidth in which your defined service operates normally aka known as baseline
+* *Measuring:* A measurement is a single act that quantifies an attribute of a part, equipment, service or process (CPU load, available memory etc.). Data measured is emitted by the monitored resources and aka telemetry. 
+* *Monitoring consumer:* The user trying to keep the service within its baseline boundaries.
++
+--
+The key to achieve that is a single control plane is usually preferred to simplify the operations for the consumer aka monitoring plane. The relevant content depends on the perspective of the consumer such as performance, costs, compliance and health. Performance in this pattern includes the following (https://github.com/uglide/azure-content/blob/master/articles/best-practices-monitoring.md[Link]):
+
+** *Health monitoring:* purpose of health monitoring is to generate a snapshot of the current health of the system so that you can verify that all components of the system are functioning as expected. This discipline includes checking for bugs and errors. 
```

Comment:
Does the health check really include checking for errors and bugs? I think it refers more to whether the system is running or whether database connections are present, for example.